game-resources/weapons/bekas.weapon

```diff
@@ -12,7 +12,7 @@
 		DamageFallOffStart = 0
 		DamageFallOffEnd = 0
 		HeadshotMultiplier = 1.2
-		Spread = 0.2
+		Spread = 0.17
```

Comment:
@the-remeko I think this is a bit too low for the shotgun, no?

Reply:
![sboxtttshotty](https://user-images.githubusercontent.com/94231804/204105576-3dc5621c-dc76-4a16-9de9-89bf5f15ea45.png)
![20221126131540_1](https://user-images.githubusercontent.com/94231804/204105593-b9839e38-4263-4b8c-b476-aa838f6fef42.jpg)

I'm close to 540 units away from the wall in both screenshots.